app/models/location.rb

```diff
@@ -2,32 +2,43 @@
 #
 # Table name: locations
 #
-#  id              :bigint           not null, primary key
-#  city_town       :string
-#  country         :string
-#  province_state  :string
-#  zipcode         :string
-#  created_at      :datetime         not null
-#  updated_at      :datetime         not null
-#  organization_id :bigint
+#  id             :bigint           not null, primary key
+#  city_town      :string
+#  country        :string
+#  locatable_type :string           not null
+#  province_state :string
+#  zipcode        :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  locatable_id   :bigint           not null
 #
 # Indexes
 #
-#  index_locations_on_organization_id  (organization_id)
-#
-# Foreign Keys
-#
-#  fk_rails_...  (organization_id => organizations.id)
+#  index_locations_on_locatable  (locatable_type,locatable_id)
 #
 class Location < ApplicationRecord
-  acts_as_tenant(:organization)
+  belongs_to :locatable, polymorphic: true
```

Comment:
This change can close this story on the kanban that hasn't been converted to an issue yet: https://github.com/orgs/rubyforgood/projects/16/views/1?filterQuery=location&pane=issue&itemId=72155107